mysql-test/suite/innodb/r/innodb_buffer_pool_resize_temporary.result

```diff
@@ -5,6 +5,11 @@ SET GLOBAL innodb_buffer_pool_size=16777216;
 CREATE TEMPORARY TABLE t1 (a INT PRIMARY KEY) ENGINE=InnoDB;
 INSERT INTO t1 SELECT seq FROM seq_1_to_200;
 SET GLOBAL innodb_buffer_pool_size=8388608;
```

Comment:
The test result shouldn't change. The form or timing of the `innodb_buffer_pool_size_status` hasn't changed. Either the test is being abnormally slow, or the fix 84b8f529c714687ff7d158caf92d08fc935146d5 needs to be revised (unrelated to this change).

I can retrigger the windows test until it passes. But we'll keep an eye on this test for repeated fajilures.

Reply:
Thanks!
> Either the test is being abnormally slow, or the fix https://github.com/MariaDB/server/commit/84b8f529c714687ff7d158caf92d08fc935146d5 needs to be revised (unrelated to this change).

I agree. When running the test individually on my own machine, I've got the same error.

Reply:
Yes. it seems quite persistent. I'll take a look sometime.

Reply:
I think I found the problem.

Currently, the test expects `INNODB_BUFFER_POOL_RESIZE_STATUS` to be `Completed resizing buffer pool.` The actual value is `Resizing other hash tables.`, which is set [here](https://github.com/MariaDB/server/blob/252e7e4aa5540c2995f6d181bf81756429488852/storage/innobase/buf/buf0buf.cc#L1917). That's the last time this status value is set, which does not reflect the reality that the resizing is done.

I made slightly more changes, by replacing one call of `ib::info` to `buf_resize_status`. This removes the redundant messages, and also improves the usefulness of `INNODB_BUFFER_POOL_RESIZE_STATUS`, because it now contains the before and after value when queried.

I've also made adjustments to a few tests affected. I think I got them all, but we'll see.

Reply:
Thanks. We'll see how @dr-m feels about changing these now that 10.8 is GA. But there's a few options if not.

Windows builder is taking a hopefully short vacation.